docs/book/v3/application-integration/form-element-manager-mvc.md

```diff
@@ -0,0 +1,283 @@
+# Usage of Form Element Manager in a laminas-mvc Application
+
+INFO:
+The following examples show the usage of the form element manager in a laminas-mvc based application.
+All **the basics of the form element manager** [can be found in a separate section](../form-element-manager.md).
+
+## Using the Form Element Manager in a Controller
+
+### Create Controller
+
+[Create a controller class](https://docs.laminas.dev/laminas-mvc/quick-start/#create-a-controller) and inject the form element manager via the constructor, e.g. `module/Album/Controller/AlbumController.php`:
+
+```php
+namespace Album\Controller;
+
+use Laminas\Form\FormElementManager;
+use Laminas\Mvc\Controller\AbstractActionController;
+
+final class AlbumController extends AbstractActionController
+{
+    public function __construct(
+        public readonly FormElementManager $formElementManager
```

Comment:
I'd recommend `private` here; we should not recommend `public` unless there's a really good reason.